pkg/serving/schema_handling.go

```diff
@@ -0,0 +1,31 @@
+package serving
+
+import (
+	"errors"
+	"fmt"
+	"github.com/knative/serving/pkg/client/clientset/versioned/scheme"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func UpdateGroupVersionKind(obj runtime.Object, gv schema.GroupVersion) error {
```

Comment:
exported functions with a comment is helpful, `/lint` prow command will pick this up.